src/components/Main/Main.scss

```
@@ -0,0 +1,45 @@
+@import "../../utils/variables.scss";
+
+.main {
+  position: relative;
+  z-index: 2;
+  border-top: 1px solid $main-border-color;
+}
+
+.toggle-all {
+  width: 1px;
+  height: 1px;
+  border: none;
+  opacity: 0;
+  position: absolute;
+  right: 100%;
+  bottom: 100%;
+
+  & + label {
+    width: 60px;
+    height: 34px;
+    font-size: 0;
+    position: absolute;
+    top: -52px;
+    left: -13px;
+    -webkit-transform: rotate(90deg);
+    transform: rotate(90deg);
+
+    &:before {
+      content: "❯";
+      font-size: 22px;
+      color: $main-border-color;
+      padding: 10px 27px;
+    }
+  }
+
+  &:checked + label:before {
+    color: $toogle-all-color;
+  }
+
+  @media screen and (-webkit-min-device-pixel-ratio:0) {
+    & {
+      background: none;
+    }
```

The same as above, need to fix it everywhere
```suggestion
    background: none;
```